src/org/labkey/test/components/react/FilteringReactSelect.java

```diff
@@ -162,6 +162,8 @@ public FilteringReactSelect filterSelect(String value, Locator elementToWaitFor)
 
     private List<WebElement> setFilter(String value)
     {
+        if (!isOpen())
+            open();
```

Comment:
I don't think this check is needed. `open` checks `isExpanded`. The real question is why are there separate `isOpen` and `isExpanded` methods and do they do different things?

Reply:
`isOpen `checks to see if the component is decorated with the `select-input__control--menu-is-open` class, `isExpanded` looks for the state of the selectMenu, pauses the check if it's loading, then returns whether or not the selectMenu is displayed or if `isOpen `returns true

If anything, isExpanded is the more-robust one